frontend/src/employee-frontend/components/ChildInformation.tsx

```diff
@@ -142,6 +145,9 @@ const layouts: Layouts<typeof components> = {
     },
     { component: 'assistance', open: false },
     { component: 'applications', open: false },
+    ...(featureFlags.childIncomeEnabled
+      ? [{ component: 'income' as keyof typeof components, open: false }]
```

Comment:
Using `as keyof typeof components` turns off type checking so using `as const` is preferable.

Reply:
Wau, fixed